### ISSUE TITLE
"module 'dask_xgboost' has no attribute * erors"

### ISSUE THREAD

**Issue by DKorman** [dask/dask-xgboost#53]
Hello,

I am experiencing strange behaviour when trying to use dask-xgboost. I am on Windows 10 machine and am using Python 3.7. I am trying to test dask on a local machine first with a local cluster.

I have no errors when importing dask_xgboost, but when I try to call any of it's functions I get a no attribute error. For some reason I am unable to use this package.
Two examples that throw the error:
1) bst = dask_xgboost.train(client, params, X_train, y_train, num_boost_round=10) --> no attribute train
2) model = dxgb.XGBClassifier() --> no attribute Xgb

This is probably something silly and perhaps Windows related.

Thanx for any help!

**Comment by TomAugspurger** [comment]
Most likely it's not installed properly. How'd you install dask-xgboost?

**Comment by DKorman** [comment]
I was using the GUI of pycharm to install it with the Conda package manager.

**Comment by TomAugspurger** [comment]
Can you create a new conda environment and post the output of installing
dask-xgboost?

**Comment by DKorman** [comment]
This is the command line output after installing dask-xgboost using anaconda prompt:

Package xgboost conflicts for:
dask-xgboost -> xgboost
Package vc conflicts for:
python=3.7 -> vc[version='14.*|>=14,<15.0a0|>=14.1,<15.0a0']
Package distributed conflicts for:
dask-xgboost -> distributed[version='>=1.16.0']
Package openssl conflicts for:
python=3.7 -> openssl[version='>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a']
Package sqlite conflicts for:
python=3.7 -> sqlite[version='>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.28.0,<4.0a0|>=3.29.0,<4.0a0']
Package pip conflicts for:
python=3.7 -> pip
Package dask conflicts for:
dask-xgboost -> dask[version='>=0.14.0']


This is output fresh after new environment, without dask installed.

**Comment by TomAugspurger** [comment]
Yeah, that didn't install correctly. I'm not entirely sure why though.

Is that from the conda-forge channel? May need to make a new release / build.

**Comment by DKorman** [comment]
Hmm, I didnt specify to use forge channel. Does it redirect to it if it doesnt find the item on the primary channel?

**Comment by TomAugspurger** [comment]
Depends on your conda config. `conda config --show-sources `
